app/src/main/assets/india.ics

```diff
@@ -215,60 +215,60 @@ END:VEVENT
 BEGIN:VEVENT
 SUMMARY:Buddha Purnima
 UID:ind_27
-DTSTART;VALUE=DATE:20160521
-DTEND;VALUE=DATE:20160522
+DTSTART;VALUE=DATE:20210526
+DTEND;VALUE=DATE:20210527
 STATUS:CONFIRMED
 RRULE:FREQ=YEARLY;INTERVAL=1
 EXDATE:20170522}
 END:VEVENT
 BEGIN:VEVENT
 SUMMARY:Raksha Bandhan
 UID:ind_28
-DTSTART;VALUE=DATE:20160705
-DTEND;VALUE=DATE:20160706
+DTSTART;VALUE=DATE:20210822
+DTEND;VALUE=DATE:20210823
 STATUS:CONFIRMED
 RRULE:FREQ=YEARLY;INTERVAL=1
 EXDATE:20170706}
 END:VEVENT
 BEGIN:VEVENT
 SUMMARY:Parsi New Year
 UID:ind_29
-DTSTART;VALUE=DATE:20160817
-DTEND;VALUE=DATE:20160818
+DTSTART;VALUE=DATE:20220819
+DTEND;VALUE=DATE:20220820
 STATUS:CONFIRMED
 RRULE:FREQ=YEARLY;INTERVAL=1
 EXDATE:20180818}
 END:VEVENT
 BEGIN:VEVENT
 SUMMARY:Janmashtami
 UID:ind_30
-DTSTART;VALUE=DATE:20160825
-DTEND;VALUE=DATE:20160826
+DTSTART;VALUE=DATE:20210831
+DTEND;VALUE=DATE:20210901
 STATUS:CONFIRMED
 RRULE:FREQ=YEARLY;INTERVAL=1
 EXDATE:20170826}
 END:VEVENT
 BEGIN:VEVENT
 SUMMARY:Solar Eclipse [Annular]
 UID:ind_31
-DTSTART:20160901T070800Z
-DTEND:20160901T071200Z
+DTSTART;VALUE=DATE:20160901
+DTEND;VALUE=DATE:20160901
```

Comment:
should end with 0902 no? If it lasts only the first day, just like other events

Reply:
Solar Eclipse last for a couple of hours only that's why. Anyway, I will try to find the exact time of the eclipse and fix it up. Thanks.

Reply:
right, should be good like that. If its just a few hours, it should have DTSTART and DTEND labels too, for start and end time. You can see some example if you try sharing an event. Anyway, this looks ok, thanks